pom.xml

```diff
@@ -81,14 +77,20 @@
     <dependency>
       <groupId>me.clip</groupId>
       <artifactId>placeholderapi</artifactId>
-      <version>2.8.7</version>
+      <version>2.10.9</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>me.glaremasters</groupId>
       <artifactId>MultiEconomy</artifactId>
-      <version>1.0.2-BETA</version>
-      <scope>provided</scope>
+      <version>1.0.6-RELEASE</version>
+      <scope>system</scope>
+      <systemPath>${project.basedir}/libs/MultiEconomy-1.0.6-RELEASE.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>org.jetbrains</groupId>
+      <artifactId>annotations</artifactId>
+      <version>19.0.0</version>
```

Comment:
From what I know does latest PAPI release not require this.

Reply:
Yeah I know it's not required because PlaceholderAPI already has it as dependency. I'll remove it.